bootstrap/kubeadm/api/v1beta1/kubeadmconfig_types.go

```diff
@@ -675,19 +675,42 @@ type DiskSetup struct {
 	Filesystems []Filesystem `json:"filesystems,omitempty"`
 }
 
+// DiskLayout represents an array of partition specifications
+type DiskLayout []PartitionSpec
+
+// PartitionSpec defines the size and optional type for a partition
+type PartitionSpec struct {
+	// Percentage of disk that partition will take (1-100)
```

Comment:
Godoc must start with the serialised version of the field name
```suggestion
	// percentage of disk that partition will take (1-100)
```